desktop/com.github.xournalpp.xournalpp.appdata.xml

```diff
@@ -75,6 +75,7 @@
     <!-- Could be automated more:
 	 `git tag -l | grep "^[0-9]*\.[0-9]*\.[0-9]*$`"
     -->
+    <release date="2022-02-13" version="1.1.1+dev" />
```

Comment:
That looks premature.
```suggestion
```

Reply:
Development versions are added by the version bump:
https://github.com/xournalpp/xournalpp/blob/b1b1b2d7023d2bb4b88922b34afbfb67f6c0b652/scripts/release_helper.sh#L167-L176